src/renderer/VideoPlayer.tsx

```diff
@@ -287,6 +287,25 @@ export const VideoPlayer = (props: IProps) => {
     }
   };
 
+  // By default the window hijacks media keys even when
+  // the window isn't focused or it is minimized
+  // so we override the action handlers
+  useEffect(() => {
+    ipc.on('window-focus-status', (arg: unknown) => {
```

Comment:
I can't quite remember if this is required or not, but I think you might need to return a function from this useEffect hook to remove the listener, like we do here: https://github.com/aza547/wow-recorder/blob/main/src/renderer/containers/ApplicationStatusCard/Status.tsx#L214? 

Might be this is fine without it, but have definitely seen warnings in the console in dev mode when I've forgot this sort of thing before. Happy enough if you think it's not required but worth checking. IIRC the warning comes after 10+ listeners are added for the same event, so I think if you just flick between a bunch of videos 10+ times you should either see it or not.